src/make_trade.ts

```diff
@@ -0,0 +1,82 @@
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+import { HardhatEthersHelpers } from "@nomiclabs/hardhat-ethers/types";
+import ERC20 from "@openzeppelin/contracts/build/contracts/ERC20.json";
+import { TokenInfo, TokenList } from "@uniswap/token-lists";
+import { BigNumber, Contract } from "ethers";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import fetch from "node-fetch";
+
+async function makeTrade(
+  tokenListUrl: string,
+  { ethers }: HardhatRuntimeEnvironment
+): Promise<void> {
+  const [trader] = await ethers.getSigners();
+
+  const all_tokens = (await fetchTokenList(tokenListUrl)).tokens;
+  const tokens_with_balance = await tokensWithBalance(
+    all_tokens,
+    trader,
+    ethers
+  );
+  if (tokens_with_balance.length === 0) {
```

Comment:
Rust `snake_case` :rofl:

Very nit-picky of me, but I'm not a huge fan of the inconsistent casing for variables.

Reply:
Changed